src/pages/SubmissionConfig.test.js

```diff
@@ -1078,11 +1094,7 @@ describe('SubmissionConfig records selector', () => {
   })
 })
 
-describe('SubmissionConfig inputs/outputs definitions', () => {
-  // SubmissionConfig component uses AutoSizer to determine the right size for table to be displayed. As a result we need to
-  // mock out the height and width so that when AutoSizer asks for the width and height of "browser" it can use the mocked
-  // values and render the component properly. Without this the tests will be break.
-  // (see https://github.com/bvaughn/react-virtualized/issues/493 and https://stackoverflow.com/a/62214834)
```

Comment:
small nit: it might be worth keeping these comments as they are useful in understanding why we set `offsetHeight` and `offsetWidth`. Although this gets repeated in all `describe()` in this file so I am not sure how to remove the redundancy and display it just once, maybe somewhere at the top of the file?